src/idiom248ex.fz

```diff
@@ -0,0 +1,15 @@
+ex248 is
+  construct_float(s bool, m f64, e i32) =>
```

Comment:
`2 ** e` may overflow as soon as e > 16. I'd rather use `u32` since negative numbers are not allowed anyway and convert to f64 before doing the power of 2.